pkg/asset/manifests/network-operator.go

```diff
@@ -1,72 +1,117 @@
 package manifests
 
 import (
+	"os"
+	"path/filepath"
+
 	"github.com/ghodss/yaml"
 	"github.com/pkg/errors"
 
 	"github.com/openshift/installer/pkg/asset"
 	"github.com/openshift/installer/pkg/asset/installconfig"
 
-	tectonicnetwork "github.com/coreos/tectonic-config/config/tectonic-network"
+	netopv1 "github.com/openshift/cluster-network-operator/pkg/apis/networkoperator/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+var (
+	noCrdFilename = filepath.Join(manifestDir, "cluster-network-01-crd.yml")
+	noCfgFilename = filepath.Join(manifestDir, "cluster-network-02-config.yml")
+)
+
 const (
-	defaultMTU         = "1450"
-	noCfgFilename      = "network-operator-config.yml"
-	noManifestFilename = "network-operator-manifests.yml"
+
+	// we need to manually create our CRD first, so we can create the
+	// configuration instance of it
+	netConfigCRD = `
+---
```

Comment:
Most of our YAML templates don't use an explicit document-start `---`:

```console
$ git grep apiVersion pkg/asset | wc -l
43
$ git grep -- --- pkg/asset | wc -l
16
```

Can we drop this line to try and consolidate around that approach?